### ISSUE TITLE
As an end user, I want to be able to disable some ZenHub features

### ISSUE THREAD

**Issue by Rican7** [ZenHubIO/support#192]
Currently, at my organization, we love to use some of the features that Zenhub provides. The board and pipeline integration is wonderful, however some of the features like the "voting"/+1'ing of comments or the file uploading are features that we prefer to not use.

Some of these features may even interfere with the current feature set provided by GitHub natively (for example the cross-over in upload functionality, or the way the uploader takes a chunk out of the comment box's width).

A simple toggle of these features in the extension's native options area, or even organization wide, would be ideal.

(related to #177)

**Comment by mcos** [comment]
:+1:

**Comment by azenMatt** [comment]
Hi @Rican7, 

Thank you for the feedback - we will take a look into making these features optional. 


> Some of these features may even interfere with the current feature set provided by GitHub natively (for example the cross-over in upload functionality

To provide some additional information around this - ZenHub only covers the files that GitHub does not support. If you pass an image file (which are the only files GitHub supports) into the ZenHub upload box, it will simply pass it on to GitHub to handle. 

It is only when you upload non-GH-supported file types that ZenHub will do the uploading and storage for you.

**Comment by Rican7** [comment]
@azenMatt Thank you for the quick response and acknowledgement. I appreciate it. :)

Yea, that's a really smart way of handling the uploads, to make sure to not interfere with their native functionality, however the uploader "click or drag" button still takes up an unnecessary amount of horizontal commenting space.

Either way, it'd be nice to disable certain features to deter users in an org from accidentally using a feature that an org has another tool for already (like Dropbox, etc).

**Comment by p-run** [comment]
+1 to disable the uploader.

**Comment by andyhqtran** [comment]
My organization would love the ability to disable the features we don't need.

For example disabling the estimation.

**Comment by ccpmark** [comment]
We would love the ability to disable some of these features, especially the upload box which takes up a good portion of the comment field!

It seems to be common among other Chrome extensions to include a settings page allowing users to enable/disable certain features to their liking.

**Comment by aensley** [comment]
Agreed. Since GH has added the ability to upload more file types now, the ZenHub uploader is completely redundant. We would also turn off the Burndown, Estimation, and ToDos.

**Comment by npcode** [comment]
I need this too. +1 feature of zenhub was awesome but now github itself supports.

**Comment by azenMatt** [comment]
Hi @npcode - On the latest version of ZenHub, you will notice that you are no longer able to +1 issues as we have removed it to allow you to use GitHub reactions :smile:

**Comment by npcode** [comment]
@azenMatt Great! Thanks for letting me know!

**Comment by theofidry** [comment]
Same here: I love ZenHub and use it for some projects, but it's way too intrusive to have it "enabled" for any GitHub project.
